src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-bar-form search-bar-form--big"
```

Comment:
Don't forget to check the `checklist`, there you can find tips on how to maintain good code and more.
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/54860641/c0486146-aa76-480a-a534-7b58df79a07d)